test/samples/nestedRepeated/input.json

```diff
@@ -1,18 +1,22 @@
 {
-	"id": "http://yourdomain.com/schemas/myschema.json",
-	"description": "Example description",
-	"type": "object",
-	"properties": {
-		"names": {
-			"type": "array",
-			"items": {
-				"type": "object",
-				"properties": {
-					"first": {
-						"type": "string"
-					}
-				}
-			}
-		}
-	}
+    "id": "http://yourdomain.com/schemas/myschema.json",
+    "description": "Example description",
+    "allOf": [
```

Comment:
We dont need to add allOf here as we have other examples that do this, can we revert this please.